LICENSE

```diff
@@ -1,3 +1,8 @@
+SPDX-License-Identifier: APACHE-2.0-only
```

Comment:
@gichan-jang , Could you update the LICENSE file of the nnstreamer repository such as this change? Thanks in advance. 
* https://github.com/nnstreamer/nnstreamer/blob/main/LICENSE

Reply:
Why SPDX header is at the LICENSE statement??

Reply:
Reference: https://github.com/spdx/license-list-data/blob/master/text/LGPL-2.1-only.txt

Reply:
@leemgs
In my opinion, specifying SPDX identifier in the license file is unnecessary, and if necessary, let's add a description to the bottom of the file.
For example,
```
----------------------------------------------------------------
Note:
Individual files contain the following tag instead of the full license text.

              SPDX-License-Identifier: APACHE-2.0-only

This enables machine processing of license information based on the SPDX
License Identifiers that are here available: http://spdx.org/licenses/

```

Refer: https://spdx.dev/wp-content/uploads/sites/41/2020/04/using_spdx_license_list_short_identifiers.pdf
13th page.

Reply:
I added  the description to the bottom of the LICENSE file. Please check it.